lib/util/util.js

```diff
@@ -33,6 +33,13 @@ const sanitizeOdataIdentifier = (name) =>
   name.replace(/^([^a-z_])/i, '_$1').replace(/([^a-z0-9_]+)/gi, '_');
 
 
+// Decodes html entities into plain text
+// greedy approach, only converting entities that are return by xlsform to xform conversion process
+const decodeHtml = (input) =>
+  input.replace(/&amp;/g, '&')
+    .replace(/&lt;/g, '<')
+    .replace(/&gt;/g, '>');
```

Comment:
Elsewhere we use this `decodeXML()` function:

https://github.com/getodk/central-backend/blob/321ac47d182e64cfffa9fb9e4ed421c5a614cb4a/lib/data/schema.js#L24

Would it be possible to use that instead of writing our own `decodeHtml()` function?